src/components/About.js

```diff
@@ -1,18 +1,38 @@
-import React from "react";
+import React, { useRef, useState, useEffect } from "react";
 import { useHistory } from "react-router-dom";
 
 import textStyles from "./TextSection/text-section.module.css";
 import imgStyles from "../css-modules/photo-text-section.module.css";
 import newsletterStyles from "../components/Newsletter/newsletter.module.css";
 
+const cx = require("classnames");
+
 export default function About(props) {
+  const placeholder = useRef();
+  const [showImage, setShowImage] = useState(false);
```

Comment:
Remember these when [naming cheatsheet](https://github.com/kettanaito/naming-cheatsheet) for your career. For real, bookmark it. It makes the code much easier to read.

In this case `isImgVisible` is a better name because it shows right away that it's a bool.